### ISSUE TITLE
Consider: User extensibility

### ISSUE THREAD

**Issue by apbendi** [cloudmine/CMHealthSDK-iOS#56]
Should we provide a method for developers to add additional properties to the user profile? How would we do this? Expose the user profile object and allow it to be extended? This is not necessarily a requirement for a Beta, or even 1.0 release.

**Comment by Behrad3d** [comment]
I think it would be necessary to have user extensibility. Because we are dealing with health care applications. Developers are required to keep patient information separate from study data. For example in our case, we have three additional parameters that we must keep in user profile as they are patient information. Currently the SDK does not let developer do it unless by modifying the internal module.

**Comment by apbendi** [comment]
Agreed @Behrad3d, we'll be working to make this a part of the framework before an official release. In the meantime, if you're playing with CMHealth or considering using it for research app or other context, we'd love to hear about your use case. Feel free to reach out and keep the feedback coming! Thanks :)
